tests/test-helper.js

```diff
@@ -1,12 +1,12 @@
 import Application from 'dummy/app';
```

Comment:
Updated the file to match the one from the addon blueprint again. Makes it a bit easier for future blueprint updates.